lib/puppet/util/run_mode.rb

```diff
@@ -55,11 +55,19 @@ def which_dir( system, user )
 
     class UnixRunMode < RunMode
       def conf_dir
-        which_dir("/etc/puppet", "~/.puppet")
+        which_dir("/etc/puppetlabs/agent", "~/.puppet")
       end
 
       def var_dir
-        which_dir("/var/lib/puppet", "~/.puppet/var")
+        which_dir("/opt/puppetlabs/agent/cache", "~/.puppet/var")
+      end
+
+      def run_dir
+        which_dir("/var/run/puppetlabs", "~/.puppet/var/run")
+      end
+
+      def log_dir
+        which_dir("/var/log/puppetlabs/agent", "~/.puppet/var/log/")
```

Comment:
@nwolfe the AIO path changes have `logdir` and `rundir` no longer falling under `vardir`, with the specified directories being the new defaults. Is there a reason you don't want to use the new defaults? In the case of `logdir` being `/var/log/puppetlabs/agent`, from what I've heard we know it's a bit odd, but it's still the decided path for AIO.